conanfile.txt.in

```diff
@@ -7,12 +7,13 @@ sdl2/2.0.10@bincrafters/stable
 sdl2_ttf/2.0.15@bincrafters/stable
 sdl2_image/2.0.5@bincrafters/stable 
 bzip2/1.0.8@conan/stable
+libmad/0.15.1b
```

Comment:
Why are you adding `libmad` as a requirement?

Reply:
Shouldn't conan handle dependencies for us?

Reply:
that's a workaround. SDL_mixer would use the (broken) bincrafters libmad library, whilst the CCI lib is already fixed.